### ISSUE TITLE
Error with expo React Native

### ISSUE THREAD

**Issue by UsmanAsif193** [plaid/react-native-plaid-link-sdk#541]
I am trying to use this sdk with react native but its not working and showing me an error of null

**Comment by dtroupe-plaid** [comment]
Hi @UsmanAsif193 

It's hard to provide any meaningful feedback when little to no information is provided about the issue. This SDK currently does not support Expo. I don't think it would take much work to add Expo support and we welcome a pull-request to add this feature. 

Since Expo is not supported right now I am going to close this issue.

**Comment by UsmanAsif193** [comment]
Can you make it support expo or tell me how to make it work with expo?

**Comment by dtroupe-plaid** [comment]
We have expo support in our backlog, but we currently do not have a timeline. 

Using this SDK in an expo app would be the same as using any other React Native bare app library in Expo. Again, Expo is not supported so we currently do not provide recommendations on how to do this.

**Comment by ChanukaSandeepa** [comment]
I've done native configs for Expo and the popup is opened only when the token is expired. Otherwise it just shows the loading and then it closes right away.

**Comment by jeanniton-mnr** [comment]
I am implementing Plaid inside an Expo-managed react-native application.

I use the `create` and `open` API to create an authentication / access token.

When I call `open`, nothing happens.

I do not even have a log.

When are you considering supporting an expo-managed app?

**Comment by phoenixy1** [comment]
Currently there is no planned support for Expo. Our current recommendation for all frontends without official SDK support is to use [Hosted Link](https://plaid.com/docs/link/hosted-link/).

**Comment by jeanniton-mnr** [comment]
> Currently there is no planned support for Expo. Our current recommendation for all frontends without official SDK support is to use [Hosted Link](https://plaid.com/docs/link/hosted-link/).

Thank you, @phoenixy1 !

I have done just that.

I have hosted a Plaid Link page. I then use React-Native WebView and an IFrame to display the Plaid page within the React native or web app. Finally, my React-Native app uses the `onMessage` API to communicate with the (hosted) webpage.

I hope to have the time one day to write a tutorial on this sample approach to guide others.

I have gained so much using Expo app management (when my app is a web-managed app) that I would just like to let that go.

If anyone insists, I will prioritize publishing a tutorial, ASAP.
